nistats/design_matrix.py

```diff
@@ -105,38 +114,43 @@ def _blank_drift(frame_times):
     return np.reshape(np.ones_like(frame_times), (np.size(frame_times), 1))
 
 
-def _make_drift(drift_model, frame_times, order=1, hfcut=128.):
+def _make_drift(drift_model, frame_times, order=1, period_cut=128.):
```

Comment:
This was a mistake: 128 stands for 128s, it is thus actually a period, not a frequency.